packages/vuetify/src/util/__tests__/mergeData.spec.ts

```diff
@@ -34,8 +34,8 @@ describe('mergeStyles', () => {
   it('should merge styles', () => {
     const cUndefined = undefined
     const cEmptyString = ''
-    const cString = 'foo: bar; fizz-buzz: 10px'
-    const cObject = { foo: 'bar', fizzBuzz: '10px' }
+    const cString = 'foo: bar; fizz-buzz: 10px; background: var(--background)'
+    const cObject = { foo: 'bar', fizzBuzz: '10px', background: 'var(--background)' }
```

Comment:
This is good, but I meant defining them too

Reply:
it's just a string parsing, nothing really to do with css variables

Reply:
https://github.com/vuetifyjs/vuetify/blob/44f27458e7da1c2e106a5ad1ac3f798317abb161/packages/vuetify/src/util/mergeData.ts#L15